linera-service/src/project.rs

```diff
@@ -80,8 +81,17 @@ impl Project {
             .current_dir(&self.root)
             .spawn()?
             .wait()?;
-        if !cargo_test.success() {
-            bail!("tests failed")
+        if !unit_tests.success() {
+            bail!("unit tests failed")
+        }
+        let integration_tests = Command::new("cargo")
+            .arg("test")
+            .args(["--target", CURRENT_PLATFORM])
```

Comment:
Wouldn't that be the default anyway, if there's no `--target` argument?

Reply:
Good point but not in this case, our templates add a `config.toml` with the following content:
```toml
[build]
target = "wasm32-unknown-unknown"
```

This is so that developers can just `cargo build` as they're building without having to specify the target every time.